config/configload/validate.go

```diff
@@ -306,3 +312,21 @@ func validMethods(methods []string, attr *hclsyntax.Attribute) error {
 
 	return nil
 }
+
+// validateBackendTLS determines irrational backend certificate configuration.
+func validateBackendTLS(block *hclsyntax.Block) error {
+	validateCertAttr := bodySyntax.AttributeByName(block.Body, "disable_certificate_validation")
```

Comment:
It would be more code in this place but you are right for the func itself. Otherwise i have to `val, ok` and check ok every time. The attr may not be there.